Source/Charts/Formatters/DefaultValueFormatter.swift

```diff
@@ -22,69 +23,65 @@ open class DefaultValueFormatter: NSObject, ValueFormatter
     
     @objc open var block: Block?
     
-    @objc open var hasAutoDecimals: Bool = false
+    @objc open var hasAutoDecimals: Bool
```

Comment:
why remove default value here but initialize later in init() many times?

Reply:
Because it has different values based on which initializer. This way it's not assigned twice.

Reply:
alright, makes sense then.